templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: <a href="{% url "taxi:driver-detail" user.id %}">{{ user.username }}</a></li>
+    <li><a href="{% url "logout" %}">Logout</a></li>
+  {% else %}
+    <li><a href="{% url "login" %}">Login</a></li>
+  {% endif %}
+  <br>
```

Comment:
Ensure that you use 2 whitespaces indentations in your `.html` files as per the checklist requirement.